lightning/src/routing/scoring.rs

```diff
@@ -683,7 +683,7 @@ impl<L: Deref<Target = u64>, T: Time, U: Deref<Target = T>> DirectedChannelLiqui
 		}
 	}
 
-	/// Computes and combines the liquidity and amount penalties.
+	/// Computes the liquidity penalty from the core and amount penalty multipliers.
```

Comment:
Let's rename the function `liquidity_penalty_msat ` and just say "from the parameterized multipliers" here.

Reply:
I also find the 'core' terminology a bit confusing here, but fine by me if it will be changed soon anyways. 🤷‍♂️

Reply:
Sure, I'd see at least drop the "core" terminology as I also find that not obvious.

Reply:
Could use "base" and "proportional" if we're trying to mirror `fee_base_msat` and `fee_proportional_millionths`.

Reply:
Ah, right. Yeah, just updating the comment should be fine. Could change "base" to "fixed" to avoid overloading "base", but I'm somewhat indifferent on that currently.